### DISCUSSION TITLE
Importer fails when using Docker implementation on non-standard port

### DISCUSSION THREAD

**Discussion by ll3N1GmAll** [firefly-iii/firefly-iii#9414] [Firefly III]
Importer fails when using Docker implementation on non-standard port. 
I tried navigating to the importer on port 81 from a remote PC (accessing via IP) and locally via localhost and 127.0.0.1. Importer operations try to navigate to URLs on localhost even when using this from a remote PC. This seems like a bug since I am logged in to 192.168.x.x and the Importer operation shows it is trying to go to "localhost" in the URL after clicking "submit". 
Also, importer operations try going to port 80 in the URL field even though firefly is running successfully on a non standard port.

**Comment by JC5** [comment]
Which reverse proxy is in front of the importer?

**Reply by ll3N1GmAll** [reply]
All I have is the docker deployment with the 4 docker images that come down as a part of that setup. I do not have any additional reverse proxy in front of it. I get this odd behavior even when accessing the importer directly from a browser on the server, connecting via 127.0.0.1:81.

**Comment by JC5** [comment]
I have a hard time replicating this. I've set up a basic combo at localhost:80 and localhost:81, and it works as expected.

Could you share some environment variables with me? From the data importer, I'd expect something like this:

```
FIREFLY_III_URL=http://app:8080
VANITY_URL=http://localhost
```
